rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -220,6 +232,73 @@ def test_aave_v3_deposit(database, ethereum_inquirer, ethereum_accounts) -> None
     assert events == expected_events
 
 
+@pytest.mark.vcr
+@pytest.mark.parametrize('ethereum_accounts', [['0x8abad97dBdBE16D043a5df66cf9E120D13708a3F']])
+def test_aave_v3_deposit_with_interest(database, ethereum_inquirer, ethereum_accounts) -> None:
+    tx_hash = deserialize_evm_tx_hash('0xd5dc3d24da9a97957743c09b8655154ffedb3cb40325b698b1159aa2fe9cf166')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1723640027000)
+    deposit_amount, interest_amount, gas_fees = '0.21191208', '0.00000083', '0.000975505588598266'
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_WBTC,
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Deposit {deposit_amount} WBTC into AAVE v3',
+            counterparty=CPT_AAVE_V3,
+            address=string_to_evm_address('0x5Ee5bf7ae06D1Be5997A1A72006FE6C607eC6DE8'),
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+            asset=EvmToken('eip155:1/erc20:0x5Ee5bf7ae06D1Be5997A1A72006FE6C607eC6DE8'),
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Receive {deposit_amount} aEthWBTC from AAVE v3',
+            counterparty=CPT_AAVE_V3,
+            address=ZERO_ADDRESS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=3,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.INTEREST,
+            asset=EvmToken('eip155:1/erc20:0x5Ee5bf7ae06D1Be5997A1A72006FE6C607eC6DE8'),
+            balance=Balance(amount=FVal(interest_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Receive {interest_amount} aEthWBTC as interest earned from AAVE v3',
```

Comment:
I would use accrued rather than earned but you can also ask lefteris that has a better english than me xd

Reply:
I guess accrued is when it's not receive in our wallet yet, and earned is when it's received. But yeah, can check with ser @LefterisJP also